index.html

```diff
@@ -674,6 +674,24 @@ <h1>Common Constraints</h1>
 
     <!-- Identifiers TODO -->
 
+    <!-- Accessibility -->
+
+    <section id="common-constraints-accessibilityy">
+      <h2 id="accessibility">Accessibility</h2>
+      <p>Authors of <em>Thing Descriptions</em> must ensure that the things described by them are accessible to
+ 	users with disabilities.
+      </p>
+      <p><span class="rfc2119-assertion" id="common-constraints-a11y-1">
+	  It is MANDATORY to provide a <code>title</code> that can be automatically rendered in a non-visual way 
```

Comment:
I think, we should use here the rfc2119 terminology "SHOULD".

Reply:
does this apply to "all" title terms in the TD model or only the top level one?

Reply:
fyi we have already this kind of discussion here: https://github.com/w3c/wot-profile/issues/138